src/components/DateSelector/DateSelectorPicker.tsx

```diff
@@ -24,7 +26,8 @@ type DateSelectorPickerProps = Omit<DayPickerProps, 'children'> & {
 };
 const defaultChildren = ({ date, onOpen }: ChildrenFunctionParams) => (
   <chakra.button onClick={onOpen} px="2" type="button">
-    {date.format('DD MMM YYYY')}
+    {/* // we use locale to update date language as changing the global locale doesn't affect existing instances. */}
```

Comment:
careful, double comment here

```suggestion
    {/* We use locale to update date language as changing the global locale doesn't affect existing instances. */}
```